client/blocks/eligibility-warnings/index.tsx

```diff
@@ -189,7 +190,11 @@ function getProceedButtonText( holds: string[], translate: LocalizeProps[ 'trans
 function isProceedButtonDisabled( isEligible: boolean, holds: string[] ) {
 	const resolvableHolds = [ 'NO_BUSINESS_PLAN', 'SITE_UNLAUNCHED', 'SITE_NOT_PUBLIC' ];
 	const canHandleHoldsAutomatically = holds.every( ( h ) => resolvableHolds.includes( h ) );
-	return ! canHandleHoldsAutomatically && ! isEligible;
+
+	// If it's not eligible for Atomic transfer lets also make sure it's not already Atomic with a plan below business.
+	return (
+		! canHandleHoldsAutomatically && ! isEligible && ! isAtomicSiteWithoutBusinessPlan( holds )
+	);
```

Comment:
I think this is what's confusing me in my comment below.

Previously, the `EligibilityWarnings` was only handling the AT transfer (clicking on the "Proceed" button would always initiate the AT transfer) but now that doesn't seem to be case any longer? Are we using this component now as an upgrade nudge for sites already transferred to the AT infrastructure?

Reply:
>Are we using this component now as an upgrade nudge for sites already transferred to the AT infrastructure?

Yes, AFAIK this component has always had the ability to show upgrade nudges. We have just adjusted the logic for our use case.

Reply:
What if we keep the current behavior on the `EligibilityWarnings` component and do something like:

```js
if ( ! isAtomicSite ) {
  // render `EligibilityWarnings`
} else if ( isAtomicSiteWithoutBusinessPlan ) {
  // render `UpgradeNudge`
} 
```

Reply:
> Yes, AFAIK this component has always had the ability to show upgrade nudges. We have just adjusted the logic for our use case.

Ok, that makes sense. What confuses me is that there is now a disconnection between the `isEligibleForAutomatedTransfer` selector and the `EligibilityWarnings` component, but maybe that's a not big deal. I'll have a deeper thought at this to see if there is a better alternative.

Reply:
>What if we keep the current behavior on the EligibilityWarnings component and do something like:

Im not sure how you envisioned this but this button is de-coupled from the notices. Also both types of notices (Upgrades/Warnings/Blocks etc) are handled in the same component which is the [HoldList](https://github.com/Automattic/wp-calypso/blob/trunk/client/blocks/eligibility-warnings/hold-list.tsx)